internal/options/types.go

```diff
@@ -193,6 +195,18 @@ type ResourceOptions struct {
 	// Get specifies resource.
 	Get GetOption
 }
+type ServiceOptions struct {
```

Comment:
Ditto, all of the structures are not necessary.

Reply:
fixed